smile/src/main/java/com/fasterxml/jackson/dataformat/smile/SmileFactory.java

```diff
@@ -7,6 +7,7 @@
 import com.fasterxml.jackson.core.format.InputAccessor;
 import com.fasterxml.jackson.core.format.MatchStrength;
 import com.fasterxml.jackson.core.io.IOContext;
+import com.fasterxml.jackson.core.json.PackageVersion;
```

Comment:
No, this is wrong. Please remove. `PackageVersion` gets generated by Maven build, and we must use one that is local here and not one from another package.
(I understand why this can occur when IDE complains, if no Maven build precedes etc).